src/Command/GenerateObjectAclCommand.php

```diff
@@ -48,19 +46,10 @@ final class GenerateObjectAclCommand extends QuestionableCommand
      */
     private $aclObjectManipulators = [];
 
-    /**
-     * @var ManagerRegistry|null
-     */
-    private $registry;
-
-    /**
-     * NEXT_MAJOR: Remove $registry argument.
-     */
-    public function __construct(Pool $pool, array $aclObjectManipulators, ?ManagerRegistry $registry = null)
+    public function __construct(Pool $pool, array $aclObjectManipulators)
```

Comment:
Here you are applying directly the next major comments right?

Reply:
Yes, I added an extra commit https://github.com/sonata-project/SonataAdminBundle/pull/6501/commits/0aeede4b5cf1a5036c7578d3dd6a706431a03b6b, but I should've updated the PR description.